src/layouts/KnownIncompatibilities.astro

```diff
@@ -0,0 +1,53 @@
+---
+import Page from "./Page.astro";
+
+import rawData from "../../public/api/v1/incompatible-mods.json";
+import IncompatibleModCard from "../components/atoms/IncompatibleModCard.astro";
+
+const TYPE_TO_HUMAN = {
+	"GAME": "Breaks the game",
+	"OTHERS": "Breaks other mods",
+	"WORKAROUND": "Requires a workaround",
+	"SELF": "Non-fatal errors or broken features"
+}
+
+const props = Astro.props;
+const data = rawData.sort((a, b) => a.name.localeCompare(b.name))
+---
+<Page content={props.content} frontmatter={props.frontmatter}>
+	<p>On this page you may find all the reported incompatibilities with Quilt. If you believe this list is missing anything, please feel free to tell us <a href="https://discord.quiltmc.org">on Discord</a>.</p>
```

Comment:
I would also mention they can directly make a PR editing the list.

Reply:
Done